apps/rush-lib/src/logic/EventHooksManager.ts

```diff
@@ -21,13 +21,18 @@ export class EventHooksManager {
     this._commonTempFolder = rushConfiguration.commonTempFolder;
   }
 
-  public handle(event: Event, isDebug: boolean): void {
+  public handle(event: Event, isDebug: boolean, ignoreHooks: boolean): void {
     if (!this._eventHooks) {
       return;
     }
 
     const scripts: string[] = this._eventHooks.get(event);
     if (scripts.length > 0) {
+      console.log(`Skipping event hooks for ${Event[event]} since --ignore-hooks was specified`);
```

Comment:
I believe this should go under the `if` statement below

Reply:
... oops